.eslintrc.js

```diff
@@ -37,8 +42,7 @@ module.exports = createConfig({
     {
       files: tsFiles,
       rules: {
-        '@typescript-eslint/no-confusing-void-expression': 'off', // not yet supported by CRA
```

Comment:
I've installed the ESLint plugin so we can get the latest version without waiting for CRA. This allows us to benefit from some new rules enabled in eslint-config-galex.